src/lib/converter/plugins/GitHubPlugin.ts

```diff
@@ -82,9 +90,13 @@ class Repository {
             });
         }
 
-        out = <ShellJS.ExecOutputReturnValue> ShellJS.exec('git rev-parse --short HEAD', {silent: true});
-        if (out.code === 0) {
-            this.branch = out.stdout.replace('\n', '');
+        if (!this.revision) {
+            out = <ShellJS.ExecOutputReturnValue> ShellJS.exec('git rev-parse --short HEAD', {silent: true});
+            if (out.code === 0) {
+                this.branch = out.stdout.replace('\n', '');
+            }
+        } else if (this.revision) {
+            this.branch = this.revision;
```

Comment:
If we would take `branch`, should we also use `--branch` instead of `--revision`?